eng/fv/approaches.rst

```diff
@@ -0,0 +1,191 @@
+.. SPDX-License-Identifier: CC-BY-SA-4.0
+
+.. Copyright (C) 2022 Trinity College Dublin
+
+Formal Verification Approaches
+==============================
+
+We present here an overview of a range of formal methods and tools
+that look feasible for use with RTEMS.
+
+A key criterion for any proposed tool is the ability to deploy it
+in a highly automated manner.
+This amounts to the tool having a command-line interface that covers
+all the features we require.
+One such feature is that the tool generates output that can be
+easily transformed into the formats
+used as part of the qualification datapack reports.
```

Comment:
Too specific to the ESA project's language probably. Better to be generic "formats useful for qualification." maybe

Reply:
ok